webapp/services/pokemon.service.ts

```diff
@@ -38,28 +42,84 @@ export class PokemonService {
 	}
 
 	public retrievePokemon () {
-		let headers = new Headers({
-			'Content-Type': 'application/json'});
+		return this._http
```

Comment:
Why network this? You've already got the JSON files. Couldn't you just use them directly on the frontend and save the bandwidth?